apps/dashboard/src/lib/components/Course/components/Ceritficate/Design.svelte

```diff
@@ -92,7 +92,7 @@
           <TextArea
             rows={6}
             placeholder="a little description about the course"
-            bgColor="bg-gray-100"
+            bgColor="bg-gray-100 dark:text-black"
```

Comment:
@Chifez Is there a reason we are adding a bgColor here? What's wrong with the default?

Reply:
the default bgColor prop holds a default dark:text-white class and that is why the text area content is not visible on dark mode so i override it by adding the dark:text-black class

Reply:
but instead of that i just pushed an update where the className prop holds a tailwind class of dark:text-black instead of it being on the bgColor prop

Reply:
@Chifez 

## Dark mode
<img width="1490" alt="image" src="https://github.com/rotimi-best/classroomio/assets/31730715/8fdc0893-0e09-4594-b2e0-c93ad6497532">

## Light mode
<img width="1257" alt="image" src="https://github.com/rotimi-best/classroomio/assets/31730715/bef90fe1-774e-436e-ba5d-9031175a42b4">